docs/contributing/contributing-code/contributing-code-control-plane/running-controlplane-locally.md

```diff
@@ -216,24 +227,18 @@ If you run into issues here, please re-read the prerequisites related to install
 ### I got a "InvalidTemplate" error when deploying a bicep file
 
 > sample error message:
+
 ```json
 {
   "code": "InvalidTemplate",
-  "message": "Deployment template validation failed: 'The template language version '2.1-experimental' is not recognized.'.",
+  "message": "Deployment template validation failed: 'The template language version '2.1-experimental' is not recognized.'."
 }
 ```
 
-Pull latest of the `radius-project/deployment-engine` project.
-Run submodule update to update bicep extensibility support for extensible resources:
-
-```bash
-git submodule update --init --recursive
-```
```

Comment:
same question as above - why is this not needed?

Reply:
Submodules are removed. We are only using dependencies now.